pkg/rpc/client/rpc.go

```diff
@@ -277,3 +530,16 @@ func (c *Client) SignAndPushInvocationTx(script []byte, wif *keys.WIF, gas util.
 	}
 	return txHash, nil
 }
+
+// ValidateAddress verifies that the address is a correct NEO address.
+func (c *Client) ValidateAddress(address interface{}) (*result.ValidateAddress, error) {
```

Comment:
Why an `interface{}`? It should just be a `string`. And probably you can return only an error here, because `result.ValidateAddress` only has a bool and Address is just echoed back which is not very useful. It's a strange and useless RPC call, actually.